content/departments/people-talent/talent/internship/index.md

```diff
@@ -111,8 +111,12 @@ Wage and Hour Guidelines
 
 ## 2022 Internship Cohort
 
+### Meet the members
+
 ![Internship Cohort](https://storage.googleapis.com/sourcegraph-assets/internship%20cohort.png)
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/XQ-Omphan0A" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 ### Internship Projects
 
 Prior to the end of the internship, each intern will complete a final project in the form of a blog post and video. These projects will reflect and recap the internship experience. Prompts and details have been shared internally and you can reach out to Kemper Hamilton with any questions.
```

Comment:
Another note further down (can't comment there): The date format uses the US format, but our [company communication guidelines say to use the ISO standard (YYYY-MM-DD or MM-DD)](https://handbook.sourcegraph.com/company-info-and-process/communication/#writing).